charts/cloudnative-pg/templates/grafana-dashboard.yaml

```diff
@@ -0,0 +1,12 @@
+{{- if .Values.monitoring.grafanaDashboard.enabled -}}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: cnpg-dashboard
```

Comment:
Can you make the name configurable through the values? maybe `cnpg-grafana-dashboard` as default just to be clearer.

Reply:
Changed to:

```yaml
name: {{ .Values.monitoring.grafanaDashboard.configMapName }}
```

with the new default value:

```yaml
configMapName: "cnpg-grafana-dashboard"
```